mls-rs/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "mls-rs"
-version = "0.41.4"
+version = "0.41.5"
```

Comment:
Needs to be 0.42.0 since we merged the other CR that had breaking changes already

Reply:
Done